spec.html

```diff
@@ -113,16 +110,25 @@ <h1>`Promise.any` Reject Element Functions</h1>
       1. Set _errors_[_index_] to _x_.
       1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
       1. If _remainingElementsCount_.[[Value]] is 0, then
-        1. Let _errorsArray_ be CreateArrayFromList(_errors_).
-        1. Let _error_ be a newly created `AggregateError` object.
-        1. Perform ! CreateDataProperty ( _error_, `"errors"`, _errorsArray_ ).
-        1. Return ThrowCompletion ( _error_ ).
+        1. Return ThrowAggregateError(_errors_).
       1. Return *undefined*.
     </emu-alg>
     <p>The `"length"` property of a `Promise.any` resolve element function is 1.</p>
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-promise.any-throwaggregateerror">
```

Comment:
I think that this section shouldn't be under Promise.any. I should make it more general